src/Template/Element/advanced_search_form.ctp

```diff
@@ -357,6 +372,18 @@ echo $this->Form->create('AdvancedSearch', [
                    href="http://en.wiki.tatoeba.org/articles/show/text-search">
             <?= __('More search options') ?>
         </md-button>
+        <?php if (!(isset($isSidebar) && $isSidebar)): ?>
+            <md-button type="submit" class="md-primary" formaction="">
+                <?= __('Create a search template') ?>
+            </md-button>
+            <span>
+                <md-icon>help</md-icon>
+                <md-tooltip md-direction="top" md-delay="1000">
```

Comment:
I’ve tried "tapping" the tooltip using the in-browser mobile emulation (Ctrl+Shift+M) and it looks like `md-delay` has the annoying effect of delaying the tooltip appearance by 1 second after tapping the icon. So I suggest removing this.